scylla-cql/src/types/serialize/row.rs

```diff
@@ -96,6 +101,10 @@ macro_rules! impl_serialize_row_for_unit {
             // Row is empty - do nothing
             Ok(())
         }
+
+        fn is_empty(&self) -> bool {
```

Comment:
Add `#[inline]` here, and to other `is_empty` impls that you add. It's very simple but is not generic so it won't get inlined across crates.